cmd/ztest.c

```diff
@@ -2790,12 +2790,12 @@ ztest_io(ztest_ds_t *zd, uint64_t object, uint64_t offset)
 		err = ztest_dsl_prop_set_uint64(zd->zd_name,
 		    ZFS_PROP_CHECKSUM, spa_dedup_checksum(ztest_spa),
 		    B_FALSE);
-		VERIFY(err == 0 || err == ENOSPC);
+		ASSERT(err == 0 || err == ENOSPC);
```

Comment:
Why change from `VERIFY` TO `ASSERT` here?

Reply:
Solely for consistency with the other places this check is done.  We always build `ztest` with debugging enabled so there's really no functional change.